lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -82,6 +82,73 @@ describe('LMSFilePicker', () => {
     assert.isTrue(wrapper.exists('FakeButton[label="Authorize"]'));
   });
 
+  
+  it('xx shows the try again prompt with error details after a failed authorization attempt', async () => {
```

Comment:
Having trouble with this test.

Reply:
1. try to access a file and get an ApiError not authorized which prompts to authorize
2. click authorize
3. close the authorization window <- how to do this??
4. the original error should have changed to show the invalid domain after the failed auth attempt.